backend/lm_backend/api/schemas/license_server.py

```diff
@@ -23,39 +27,47 @@ class LicenseServerWithoutConfigIdCreateSchema(BaseCreateSchema):
     The config_id will be handled in the endpoint.
     """
 
-    host: str
-    port: PositiveInt
+    host: str = Field(..., title="Host", description="The host of the license server.")
+    port: PositiveInt = Field(..., title="Port", description="The port of the license server.")
```

Comment:
This definitely should have an upper bound (65535).

Reply:
Added a validator for it.